python/desc/imsim/skyModel.py

```diff
@@ -127,34 +155,35 @@ def addNoiseAndBackground(self, image, bandpass='u', m5=None,
         PhotometricParameters class that carries details about the
         photometric response of the telescope.  Defaults to None.
 
+        @param [in] chipName is the name of the sensor being considered,
+        e.g., "R:2,2 S:1,1".
+
         @param [out] the input image with the background and noise model added to it.
         """
+        camera = get_obs_lsstSim_camera()
```

Comment:
In the interest of not having to re-implement both `addNoiseAndBackground()` and `getNoiseModel()`, should we pass `chipName` all the way through to `getNoiseModel()` and then do all of the "central pixel to RA, Dec" calculations in `getNoiseModel()`?  Then we would only have to implement that method, which was how I originally intended the `NoiseAndBackground` functions to work.  I'm just worried about this `addNoiseAndBackground()` getting out of sync with the version in `NoiseAndBackgroundBase`.  Feel free to object if you would rather just get on with your life.

Reply:
I'm not sure I'm understanding why we'd want to move those central pixel calculations to `getNoiseModel()`.  The only data that function needs are `skyLevel` and the read noise, which are passed in the existing interface.   Since this seems to work for now, let's use the current implementation and revisit this later if some divergence with the base class arises.

Reply:
Okay.

Reply:
Go ahead and merge (assuming my review was holding that up)